### PR TITLE
task solution

### PR DESCRIPTION
**Pull request by vpanch2**
[DEMO LINK] https://vpanch2.github.io/layout_catalog/

[TEST REPORT LINK] https://vpanch2.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
+$background-color:#fff;
+$primary-text-color: #060b35;
+$secondary-text-color: #616070;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/card.scss:3]: move variables into separated file

File: src/styles/card.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/card.scss:8]: move into normalize.scss

File: src/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
+$main-link-color: #00acdc;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/header.scss:1]: move into variables.scss

File: src/styles/header.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+@font-face {
+  font-family: Roboto, sans-serif;
+  src: url();
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/header.scss:6]: why do you need font-face if you haven't loaded any custom font files?

File: src/styles/header.scss [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+body,
+div,
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/header.scss:18]: remove, it already presents in normalize file

File: src/styles/header.scss [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    display: inline;
+
+    &:first-child a {
+      margin-left: 0;
+    }
+  }
+
+  &__link:link {
+    display: block;
+    font-style: normal;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/header.scss:48]: remove redundant default properties

File: src/styles/stars.scss [hunk 7]
```diff
@@ -0,0 +1,22 @@
+$size: 16px;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/stars.scss:1]: into variables file

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
1. Layout shouldn't be broken if I add more cards
image

**Review by qoa11a — CHANGES_REQUESTED** [review]
image
The layout is still broken if I add more cards.

**Pull request closed without merging**